examples/download-local-video/download_local_video.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import asyncio
+import base64
+import pathlib
+import sys
+from datetime import datetime, timedelta
+
+from yapapi import Golem
+from yapapi.services import Service
+from yapapi.log import enable_default_logger
+from yapapi.payload import vm
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
+
+from utils import (run_golem_example, TEXT_COLOR_CYAN, TEXT_COLOR_DEFAULT, TEXT_COLOR_RED)
+
+task_finished_event = asyncio.Event()
+
+class ApiCallService(Service):
+    @staticmethod
+    async def get_payload():
+        manifest = open("manifest.json", "rb").read()
+        manifest = base64.b64encode(manifest).decode("utf-8")
+
+        manifest_sig = open("manifest.json.base64.sign.sha256", "rb").read()
+        manifest_sig = base64.b64encode(manifest_sig).decode("utf-8")
+
+        manifest_sig_algorithm = "sha256"
+
+        # both DER and PEM formats are supported
+        manifest_cert = open("foo_req.cert.pem", "rb").read()
+        manifest_cert = base64.b64encode(manifest_cert).decode("utf-8")
+
+        return await vm.manifest(
+            manifest=manifest,
+            manifest_sig=manifest_sig,
+            manifest_sig_algorithm=manifest_sig_algorithm,
+            manifest_cert=manifest_cert,
+            min_mem_gib=0.5,
+            min_storage_gib=5,
+            min_cpu_threads=1,
+            capabilities=["inet", "manifest-support"],
```

Comment:
is there a current list of VM capabilities available somewhere?

Reply:
Afaik only here https://github.com/golemfactory/yagna/blob/master/utils/manifest-utils/src/manifest.rs#L65